tests/tests/files/big_cpp.cpp

```diff
@@ -18,6 +18,7 @@
 #include <stdlib.h>
 
 #include "big_header.h"
+#include "subdir/header@with,many^strange~chars.h"
```

Comment:
The hypothetical development organization behind this file could really benefit from some coding guidelines! ;)